.travis.yml

```diff
@@ -23,20 +23,23 @@ matrix:
     - stage: r10k tests
       rvm: jruby
     - stage: r10k container tests
-      dist: bionic
+      dist: focal
       language: ruby
       services:
-        # bionic uses 18.06 but need 19.03+ for buildkit so upgrade later in relevant cell
         - docker
-      rvm: 2.6.5
+      jdk: openjdk9
+      rvm: 2.6.6
       env:
+        - DOCKER_COMPOSE_VERSION=1.25.5
         # necessary to prevent overwhelming TravisCI build output limits
         - DOCKER_BUILD_FLAGS="--progress plain"
+      before_install:
+        - sudo rm /usr/local/bin/docker-compose
+        - curl --location https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname --kernel-name`-`uname --machine` > docker-compose
+        - chmod +x docker-compose
+        - sudo mv docker-compose /usr/local/bin
       script:
-        - |
-          set -ex
```

Comment:
the `set -e` might be important here. Want to make sure travis will report a failure if one of these steps returns non-zero.

Reply:
It is important - didn't think it was because of how the steps were separated with `-`, but I was wrong. It's added back.

Thanks!